src/models/Scholarship.ts

```diff
@@ -0,0 +1,21 @@
+interface ScholarshipProps {
+  // TODO(https://github.com/beyondhb1079/s4us/issues/56):
+  // Update this to reflect the schema
+  id?: string;
```

Comment:
Is the use of the ? to check if we provided an id, and if not it doesn't create that field?

Reply:
`?` makes an [optional property](https://www.typescriptlang.org/docs/handbook/advanced-types.html#optional-parameters-and-properties) here, which means it can be null/undefined. I imagine we're going to want to do that for almost every property except the minimum required ones that must be present.